models/Reaction.js

```diff
@@ -0,0 +1,30 @@
+const { Schema, Types } = require("mongoose");
+
+const reactionSchema = new Schema({
+  reactionId: {
+    type: Schema.Types.ObjectId,
+    default: () => new Types.ObjectId(),
+  },
+
+  reactionBody: {
+    type: String,
+    required: true,
+    maxlength: 280,
+  },
+
+  createdAt: {
+    type: Date,
+    default: Date.now,
+    get: function (date) {
+      return date.toISOString();
+    },
+  },
+  username: {
+    // Is this correct?
```

Comment:
In the homework brief it didn't mention referring to the user model but this is a MUCH better way of doing things than inserting a random username that doesn't link to anywhere so I would keep it this way if I was in your position. Great work :-)